minimint-core/Cargo.toml

```diff
@@ -29,3 +29,5 @@ sha3 = "0.9.1"
 tbs = { path = "../crypto/tbs" }
 thiserror = "1.0.23"
 tracing ="0.1.22"
+hbbft = { git = "https://github.com/fedimint/hbbft" }
```

Comment:
I don't think you need the entirety of `hbbft` right? We can keep it but if it was to interfere with WASM we could directly use `threshold_crypto` (which is re-exported as `hbbft::crypto`) and impl the conversion `fn to_hbbft(self) -> ConsensusOutcome` elsewhere.

Reply:
:+1: Yeah, that was my first attempt at the implementation but I wasn't sure where to put that function.